src/app/containers/RadioPageMain/index.test.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
 import { ServiceContextProvider } from '#contexts/ServiceContext';
 import { RequestContextProvider } from '#contexts/RequestContext';
-import MediaPageMain from '.';
+import RadioPageMain from '.';
 import amharicPageData from '#data/amharic/bbc_amharic_radio/liveradio';
-import addIdsToBlocks from '../../routes/getInitialData/mediapage/addIdsToBlocks';
+import addIdsToBlocks from '../../routes/getInitialData/radioPage/addIdsToBlocks';
```

Comment:
Should `radioPage` be camelCase? I'm assuming it shouldn't make a difference

Reply:
I think `frontpage` being lower case is a mistake. It's being set as camelCase in the [new preprocessor rules](https://github.com/bbc/simorgh/pull/3998). 

We have camelCase everywhere else so seems like a good time to change it for the radioPage